Tests.Integration.Presentation.Web/Security/AccessibilityTests.cs

```diff
@@ -117,7 +117,7 @@ public async Task Token_Can_Be_Invalidated_After_Creation()
                 Assert.NotNull(requestResponse);
                 Assert.Equal(HttpStatusCode.Forbidden, requestResponse.StatusCode);
             };
-            await HttpApi.DeleteUserAsync(createdUserId);
+            await UserHelper.SendDeleteUserAsync(createdUserId);
```

Comment:
if it returns a httpresponsemessage, then remember to disposeasync and expectHttpResponse (the two hepler methods)